lightly/api/__init__.py

```diff
@@ -5,3 +5,10 @@
 
 from lightly.api.api_workflow_client import ApiWorkflowClient
 from lightly.api.api_workflow_compute_worker import ArtifactNotExist
+
+# Mock the urlencode to use doseq, so that arrays in the query are properly encoded
+from six.moves.urllib.parse import urlencode
```

Comment:
will this not be overwritten when we generate the code again?

Reply:
This is not part of the generated code. The generated code is only under `lightly/openapi_generated`